### PR TITLE
CLOUDSTACK-8582: Skipping unsuitable test cases for simulator

### PR DESCRIPTION
**Pull request by gauravaradhye**
Modifying tags for test cases which shouldn't be run on simulator (Test cases requiring download of templates/ISOs and creation of templates from snapshots)

### CODE REVIEW

File: test/integration/component/test_ps_domain_limits.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ def setupAccounts(self):
             return [FAIL, e, None]
         return [PASS, None, users]
 
-    @attr(tags=["advanced", "selfservice"])
+    @attr(tags=["advanced", "selfservice"], required_hardware="false")
```
Comment (sanju1010) [inline test/integration/component/test_ps_domain_limits.py]: Is this test needs to be run on simulator?

Comment (gauravaradhye) [inline test/integration/component/test_ps_domain_limits.py]: Yes it can run, it only concerns primary storage counts.

**Comment by sanju1010** [comment]
LGTM

**Comment by gauravaradhye** [comment]
Need a second LGTM, anyone?

**Comment by runseb** [comment]
these tests used to be required_hardware=true, and now it's false. So when are we running these tests ? Are they included in Travis runs, in which case we are loosing tests with this ?

Also does this pass pep8, flake8 etc ?

**Comment by runseb** [comment]
@gauravaradhye if you answer my questions I can give a LGTM :)

**Comment by gauravaradhye** [comment]
@runseb Sorry for delayed response, I was away for two weeks.
As said by Sanjeev, the tests were wrongly tagged. required_hardware = "true" implies test can't be run on simulator and should not be, as it requires resources other than just management server.

I have only corrected the tags in this test and did not make any other changes such fixing pep8 issues, as it would confuse the reviewer.

I hope this answers your questions, please let me know in case anything unclear.

**Comment by bhaisaab** [comment]
LGTM

**Pull request closed without merging**